### PR TITLE
SchemaGeneratorPostProcessing should be deprecated

### PR DESCRIPTION
**Pull request by bbakerman**
SchemaGeneratorPostProcessing was a previous mechanism that `graphql.schema.SchemaTransformer` is better suited to

We should not have two mechanisms for the same thing

I propose we deprecated it and remove at a later date

### CODE REVIEW

File: src/main/java/graphql/schema/idl/RuntimeWiring.java [hunk 1]
```diff
@@ -352,7 +353,11 @@ public Builder comparatorRegistry(GraphqlTypeComparatorRegistry comparatorRegist
          * @param schemaGeneratorPostProcessing the non null schema transformer to add
          *
          * @return the runtime wiring builder
+         * @deprecated This mechanism can be achieved in a better way via {@link graphql.schema.SchemaTransformer}
+         * after the schema is built
          */
+        @Deprecated
+        @DeprecatedAt(value = "2012-10-29")
```
Comment (dondonz) [inline src/main/java/graphql/schema/idl/RuntimeWiring.java:360]: Nit: 2022

File: src/main/java/graphql/schema/idl/SchemaGeneratorPostProcessing.java [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
 /**
  * These are called by the {@link SchemaGenerator} after a valid schema has been built
  * and they can then adjust it accordingly with some sort of post processing.
+ *
+ * @deprecated This mechanism can be achieved in a better way via {@link graphql.schema.SchemaTransformer}
+ * after the schema is built
  */
 @PublicSpi
+@Deprecated
+@DeprecatedAt(value = "2012-10-29")
```
Comment (dondonz) [inline src/main/java/graphql/schema/idl/SchemaGeneratorPostProcessing.java:16]: Nit: should be `@DeprecatedAt("2022-10-29")`

Comment (andimarek) [inline src/main/java/graphql/schema/idl/SchemaGeneratorPostProcessing.java:16]:
```suggestion
@DeprecatedAt(value = "2022-10-29")
```

File: src/main/java/graphql/schema/idl/RuntimeWiring.java (hunk 1 above)
Comment (bbakerman) [reply src/main/java/graphql/schema/idl/RuntimeWiring.java]: fixed

File: src/main/java/graphql/schema/idl/SchemaGeneratorPostProcessing.java (hunk 2 above)
Comment (bbakerman) [reply src/main/java/graphql/schema/idl/SchemaGeneratorPostProcessing.java]: fixed

**Review by dondonz — APPROVED** [review]

**Pull request merged**